setup.py

```diff
@@ -84,7 +84,7 @@
 extras["serving"] = ["pydantic", "uvicorn", "fastapi", "starlette"]
 extras["all"] = extras["serving"] + ["tensorflow", "torch"]
 
-extras["testing"] = ["pytest", "pytest-xdist", "timeout-decorator"]
+extras["testing"] = ["pytest", "pytest-xdist", "timeout-decorator", "psutil"]
```

Comment:
@thomwolf @LysandreJik @julien-c - Hope it's ok to add psutil for testing purposes.

Reply:
Yes that’s a robust dependency